jetty-http/src/main/java/org/eclipse/jetty/http/HttpURI.java

```diff
@@ -77,6 +103,8 @@
     String _uri;
     String _decodedPath;
 
+    boolean _ambiguousSegment;
```

Comment:
Please review copy-constructor `HttpURI(HttpURI uri)` as it does not copy this new field, nor `_user`, nor `_decodedPath`.